orderbook/src/fee.rs

```diff
@@ -260,11 +263,11 @@ impl MinFeeCalculating for MinFeeCalculator {
                 min_fee,
             )
             .await;
-        Ok((min_fee, official_valid_until))
+
+        Ok((apply_fee_factor(min_fee, self.fee_factor), official_valid_until))
```

Comment:
I think this still causes issues though right? Because now in `validate_fee`, depending on if the fee is fresh (i.e. it uses `compute_min_fee`) or not (uses `get_min_fee`) the base `fee_factor` will be applied or not.

Additionally, in the case it uses `compute_min_fee` result to validate the fee, the un-subsidized fee that is returned still has the "base" fee factor applied.